fe/fe-core/src/main/java/org/apache/doris/analysis/StreamLoadStmt.java

```diff
@@ -147,12 +141,10 @@ public Separator getLineDelimiter() {
         return lineDelimiter;
     }
 
-    @Override
```

Comment:
Why is this removed?

Reply:
I mean, why did you remove the `@Override` annotation, although Java does not mandate the use of the `@Override` annotation, but we still strongly recommend using this annotation to clarify the code intent and improve code readability and maintainability. If the `@Override` annotation is not used when implementing the interface method, the compiler will not give warnings or error prompts when the method signature has typos or parameter type mismatches, which may lead to hard-to-find errors .